packages/opds-web-client/src/components/Book.tsx

```diff
@@ -3,13 +3,15 @@ import CatalogLink from "./CatalogLink";
 import BookCover from "./BookCover";
 import BorrowButton from "./BorrowButton";
 import DownloadButton from "./DownloadButton";
-import { BookData, FulfillmentLink } from "../interfaces";
+import { BookData, FulfillmentLink, BookMedium } from "../interfaces";
 import { AudioHeadphoneIcon, BookIcon } from "@nypl/dgx-svg-icons";
```

Comment:
You don't need these in this file anymore.

Reply:
Done